.github/pull_request_template.md

```diff
@@ -0,0 +1,15 @@
+## Issue addressed
+Fixes #<issue number>
+
+## Explanation
+Explain how you addressed the bug/feature request, what choices you made and why.
+
+## Checklist
+- [ ] Updated tests or added new tests
+- [ ] Branch is up to date with `main`
+- [ ] Tests & pre-commit hooks pass
```

Comment:
we don't use pre-commit hooks yet in this repos.

Reply:
Removed